ygnmi/gnmi.go

```diff
@@ -429,11 +430,13 @@ func populateSetRequest(req *gpb.SetRequest, path *gpb.Path, val interface{}, op
 	switch op {
 	case deletePath:
 		req.Delete = append(req.Delete, path)
-	case replacePath, updatePath:
+	case replacePath, updatePath, unionreplacePath:
 		var typedVal *gpb.TypedValue
 		var err error
 		if s, ok := val.(*string); ok && path.Origin == "cli" {
```

Comment:
nit: make this switch statement

Reply:
Ugh switch doesn't allow for multiple clauses: `s, ok := val.(*string); ok && path.Origin == "cli"`
So I'll keep it for now since I think the readability is fine.